build_tools/ci/run_matmul_test.sh

```diff
@@ -65,11 +65,18 @@ fi
 IREE_COMPILE_EXE=""
 TEST_RUNNER=""
 for dir in "${IREE_INSTALL_DIR}" "${IREE_INSTALL_DIR}/bin" "${IREE_INSTALL_DIR}/tools"; do
+  echo "Looking in $dir"
   if [ -f "${dir}/iree-compile" ]; then
     IREE_COMPILE_EXE="${dir}/iree-compile"
   fi
-  if [ -f "${dir}/testing/e2e/iree-e2e-matmul-test" ]; then
-    TEST_RUNNER="${dir}/testing/e2e/iree-e2e-matmul-test"
+  if [ -f "${dir}/iree-compile.exe" ]; then
```

Comment:
because if the linux (no exe) fails then the exe will succeed. Just very broadly, I'll say that I do not want to spend a lot of time fixing this script, which we agreed should either be ported to python or absorbed into `cpu_comparison/run_test.py`.

Reply:
Makes sense